services/server/server.go

```diff
@@ -468,6 +468,18 @@ func (pc *proxyClients) getClient(address string) (*grpc.ClientConn, error) {
 	connParams := grpc.ConnectParams{
 		Backoff: backoffConfig,
 	}
+	retryPolicy := `{
+		"methodConfig": [{
+		  "name": [{"service": "containerd.services.sandbox.v1.Controller", "method": "Wait"}],
+		  "waitForReady": true,
+		  "retryPolicy": {
+			  "MaxAttempts": 10000,
```

Comment:
why 10000?

Reply:
We want it to do retries in a long time to make sure the restart time of the sandboxer will not exit the sandboxes. currently we set the max backoff to 10s and the retries 10000, so approximately we can tolerate 100000s off time of sandboxer. It can be reset to a more reasonable value, if you have a better suggestion, please leave you comment here to let us know.

Reply:
100,000 seconds is approximately 27.8 hours. I suggest add comments to clarify your consider.

Reply:
That'OK.